nautobot/ipam/forms.py

```diff
@@ -722,11 +722,14 @@ class IPAddressFilterForm(NautobotFilterForm, TenancyFilterForm, StatusModelFilt
 
 
 class VLANGroupForm(LocatableModelFormMixin, NautobotModelForm):
+    range = forms.CharField(required=False, help_text="Vlan range, default 1-4094 if left blank.")
```

Comment:
Is this explicit field definition needed here? Can we make this the default representation for a `PositiveRangeNumberTextField` instead? (https://docs.djangoproject.com/en/4.2/howto/custom-model-fields/#specifying-form-field-for-model-field)

Also, why is this `required=False`?

Reply:
This is now fixed